### DISCUSSION TITLE
✅ Apply "Exit to" User Preferences to the new Editor Top Bar > Exit to Dashboard button - [ED-14305]

### DISCUSSION THREAD

**Discussion by oscarhenao** [elementor/elementor#22828] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The Exit Strategy is no longer working when the new editor Top Bar is enabled, starting on v3.14.0, even though there are three options to select from for the Exit Strategy under User Preferences, even with the Top Bar enabled.

### Describe the solution you'd like

Bring the options back and working when the Top Bar is enabled

### Describe alternatives you've considered

As of now, it is either Top Bar or Exit Strategy, which was not the case before the top bar... The top bar is a time saver and a great complement to it is the exit strategy as it was working before v3.14.0

**Comment by nicholaszein** [comment]
Hey @oscarhenao!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
